nodejs/week1/meal_sharing_website/src/backend/routes/cheapMeals.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const router = express();
+
+const meals = require("../data/meals.json");
+const reviews = require("../data/reviews.json");
+const mealReview = meals.map(meal => {
+	const mealDataCopy = Object.assign({}, meal);
```

Comment:
nice! :) You're making a copy so you don't alter the original meal in the original array :) That's a good thought to carry into the future.